package.json

```diff
@@ -59,7 +60,7 @@
   ],
   "devDependencies": {
     "@rollup/plugin-node-resolve": "^15.1.0",
-    "@solid/community-server": "^6.1.0",
+    "@solid/community-server": "^7.0.3",
```

Comment:
Was not expecting this to be changed in this PR. Leave this at version 6. We should do library updates for features not related to your changes in a separate PR.

Reply:
Reverted back.